examples/resources/bloxone_td_security_policy/resource.tf

```diff
@@ -0,0 +1,15 @@
+resource "bloxone_td_security_policy" "example" {
```

Comment:
Please add example for other optional fields too if you can

Reply:
Added a few fields for now , once DFP is merged , we can add that too